ide/coqide/wg_Detachable.ml

```diff
@@ -54,7 +51,7 @@ class detachable (obj : ([> Gtk.box] as 'a) Gobject.obj) =
       if detached then win#misc#hide ()
       else self#misc#hide ()
 
-    method visible = win#misc#visible || self#misc#visible
+    method visible = win#misc#visible || self#misc#visible [@@ocaml.warning "-7"]
```

Comment:
Warning 7 is `Method overridden without using the "method!" keyword`, is there a reason we don't use `method!` ?